EU4ToVic2Tests/EU4WorldTests/CountryTests/EU4TechnologyTests.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Country/EU4Technology.h"
+#include "gtest/gtest.h"
+
+TEST(EU4World_Country_EU4TechnologyTests, primitivesDefaultToBlank)
+{
+	std::stringstream input;
+	const EU4::EU4Technology technology(input);
+
+	ASSERT_FALSE(technology.getAdm());
```

Comment:
Assert equals to zero for clarity (also update test names). Alternately, use an optional.